services_app/src/main/res/values-es/strings.xml

```diff
@@ -207,7 +207,7 @@
   <string name="sync_status_appname_not_supported_by_server">Servidor no está configurado para este nombre de aplicación.\nPlease verificar la configuración del servidor.</string>
   <string name="sync_status_server_missing_config_files">¿Servidor no tiene toda la configuración para la aplicacion y tablas de esta versión APK. \nDesea intentar restablecer el servidor?</string>
   <string name="sync_status_server_reset_failed_device_has_no_config_files">¿Dispositivo no tiene ninguna configuración. \nDesea sincronizar con el servidor?</string>
-  <string name="sync_status_resync_because_config_has_been_reset_error">El servidor se ha restablecido con la última vez que se sincronizó y el esquema ha cambiado. La sincronización ha sido abortada.</string>
+  <string name="sync_status_resync_because_config_has_been_reset_error">opendatakit/%1$s</string>
```

Comment:
You can remove this line as English will be used for this string if no Spanish translation is available.  This will make it easier for our partners to filter for untranslated strings when they translate strings in Crowdin.

Reply:
```suggestion
```